<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditItemDialog</class>
 <widget class="QDialog" name="EditItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>261</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Изменение данных</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/window-icon.png</normaloff>:/Resources/window-icon.png</iconset>
  </property>
  <widget class="QFrame" name="editFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="authorFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="authorLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Автор</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>521</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="themeFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="themeLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Тема</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="themeLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>521</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="phraseFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="phraseLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Фраза</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="phraseTextEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>521</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
